module/VuFind/tests/unit-tests/src/VuFindTest/View/Helper/Root/MakeLinkTest.php

```diff
@@ -0,0 +1,199 @@
+<?php
+/**
+ * makeLink view helper Test Class
+ *
+ * PHP version 7
+ *
+ * Copyright (C) Villanova University 2019.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:testing:unit_tests Wiki
+ */
+namespace VuFindTest\View\Helper\Root;
+
+use VuFind\View\Helper\Root\MakeLink;
+
+/**
+ * makeLink view helper Test Class
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:testing:unit_tests Wiki
+ */
+class MakeLinkTest extends \VuFindTest\Unit\AbstractMakeTagTest
+{
+    /**
+     * Get MakeLink helper with mock view
+     *
+     * return \Laminas\View\Helper\EscapeHtml
+     */
+    protected function getHelper()
+    {
+        $helper = new MakeLink();
+        $helper->setView($this->getViewWithHelpers());
+        return $helper;
+    }
+
+    /**
+     * Test that responds to common inputs
+     *
+     * @return void
+     */
+    public function testLink()
```

Comment:
This needs tests that verify proper escaping of 'text' and any anchor with a url.

Reply:
Are these tests not covered in the examples chosen and later with the escape test? Maybe an example would help me understand.

Reply:
Sorry, I had lost track of this. Indeed they are. I just wasn't expecting escaping tests to be separate. No problem here then.